view/next-project/src/pages/purchasereports/index.tsx

```diff
@@ -241,31 +241,31 @@ export default function PurchaseReports(props: Props) {
           <table className='mb-5 w-max table-auto border-collapse md:w-full md:table-fixed'>
             <thead>
               <tr className='border border-x-white-0 border-b-primary-1 border-t-white-0 py-3'>
-                <th className='w-fit pb-2'>
-                  <div className='text-center text-sm text-black-600'>財務局長チェック</div>
+                <th className='w-1/12 pb-2'>
+                  <div className='text-center text-sm text-black-600'></div>
                 </th>
```

Comment:
```suggestion
                <th className='w-1/12 pb-2' />
```
ここのdivタグは文字の設定をしているので削除で大丈夫だと思います。